my-teachers/snippet.html

```diff
@@ -0,0 +1,55 @@
+<script>
+
+var user,$teachersList,myArray;
+
+user = FrogOS.getUser();
+
+$teachersList = $("<div>", {
+    id: Frog.Utilities.generateSimpleId()
+});
+$teachersList.html(
+    '<b>Your teachers:</b>'
+);
+
+myArray=[];
+
+
+/// Get all the groups assoicated with logged in user
+Frog.Model.api('users.getDataInCategory', {
```

Comment:
Using the Groups api is probably faster. Consider using ```groups.getFor```. It requires ONE parameter ```identity``` which would be your user uuid